med_io/pipeline.py

```diff
@@ -38,7 +38,8 @@ def pipeline(config, dataset_image_path, dataset_label_path, dataset=None):
     patches_indices = get_fixed_patches_index(config, max_data_size, patch_size,
                                               overlap_rate=config['patch_overlap_rate'],
                                               start=config['patch_start'],
-                                              end=config['patch_end'])
+                                              end=config['patch_end'],
+                                              max_patch_num=config['max_patch_num'])
```

Comment:
add to util.convert_yaml_config() check if new key max_patch_num exists, otherwise add for backward compat:
```
if not 'max_patch_num' in config.keys():
    config['max_patch_num'] = 200
```
What is a good init number? Will this affect previous pipeline?

Reply:
Added the config check with the default value **None** because this is the default value of the get_fixed_patches_index function, the behaviour of the pipeline should therefore be the same for the config files which do not have this parameter
(util.convert_yaml_config() line 35)

Reply:
Thanks 👍